firmware/spade/src/CMakeLists.txt

```diff
@@ -2,15 +2,19 @@
 set(CMAKE_BUILD_TYPE Debug)
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-void-pointer-to-int-cast -Wno-int-to-void-pointer-cast -Wno-pointer-sign -Werror=implicit-function-declaration")
 
-add_executable(spade "${SPADE_TARGET}/main.c")
-include("${SPADE_TARGET}/CMakeLists.txt")
+if (SPADE_TARGET STREQUAL "rpi")
```

Comment:
This looks a little weird...  what are we doing here?

Reply:
fixed! was a lil stupid and forgot I could just #include it